contracts/test/LON/Burn.t.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Our repo is under MIT license, maybe our tests could also use MIT.

Reply:
Certainly! I will change these code license to MIT license in the next commit.